Mathlib/Data/Fintype/Basic.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2017 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 -/
+import Mathlib.Algebra.Ring.Hom.Defs -- TODO: This import is bogus
```

Comment:
The import was already there before. But shaking made it visible. I want this to end up in mathlib because it will would be scope-creep to fix it in this PR.

I will change it to FIXME for clarity.
```suggestion
import Mathlib.Algebra.Ring.Hom.Defs -- FIXME: This import is bogus
```